src/lib/modules/Document/DocumentKeywords.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import Overridable from 'react-overridable';
+import PropTypes from 'prop-types';
+import _get from 'lodash/get';
+
+const DocumentKeywords = ({ metadata, separator, noneMessage }) => {
+  const keywords = _get(metadata, 'keywords', []);
+  if (!keywords.length) {
+    return null;
+  }
+
+  function displayKeyword(keywordValue, keywordSource) {
+    return keywordValue && keywordSource
+      ? `${keywordValue} (${keywordSource})`
+      : keywordValue
+      ? keywordValue
+      : '';
+  }
+
+  return (
+    <Overridable
+      id="DocumentKeywords.layout"
+      {...{ metadata, separator, noneMessage }}
+    >
+      <>
+        {keywords.length > 0
+          ? keywords
+              .map(keyword => displayKeyword(keyword.value, keyword.source))
+              .join(separator)
+          : noneMessage}
+      </>
+    </Overridable>
+  );
+};
+
+DocumentKeywords.propTypes = {
+  metadata: PropTypes.object.isRequired,
```

Comment:
I would prefer to pass only the keywords list if possible, instead of the entire metadata obj.
```suggestion
  keywords: PropTypes....
```